packages/clarity-js/src/data/metadata.ts

```diff
@@ -150,8 +150,8 @@ export function consent(status = true): void {
 
 export function consentv2(consentState: ConsentState = defaultStatus, source: number = ConsentSource.API): void {
   const updatedStatus = {
-    ad_Storage: normalizeConsent(consentState.ad_Storage),
-    analytics_Storage: normalizeConsent(consentState.analytics_Storage)
+    ad_Storage: typeof consentState.ad_Storage === 'string' ? consentState.ad_Storage.toLowerCase() : consentStatus?.ad_Storage ?? Constant.Denied,
```

Comment:
Minor: Would be clear if you used parentheses as its' not clear if ?? binds here based on operator precedence.

(ypeof consentState.ad_Storage === 'string' ? consentState.ad_Storage.toLowerCase() : consentStatus?.ad_Storage) ?? Cosnent.Denied